dist/PelTec.js

```diff
@@ -44,8 +47,9 @@ export class PelTecDisplay extends DisplayWithPowerButton {
         }
 
         // define Sub areas of display
-        this.bufferArea = new DisplaySubArea(this, 520, 5, 370, 570)
+        this.bufferArea = new DisplaySubArea(this, 490, 5, 370, 570)
```

Comment:
Hm, I am surprised this had to be moved.. I though I tool all values as on original site... anyway fine with that